projects/gale/index.js

```diff
@@ -0,0 +1,74 @@
+const { sumTokensAndLPsSharedOwners, unwrapUniswapLPs } = require("../helper/unwrapLPs");
+const { staking } = require("../helper/staking");
+const sdk = require("@defillama/sdk");
+const windmillABI = require("./windmillABI.json");
+const { transformAvaxAddress, transformBscAddress, transformCronosAddress, transformPolygonAddress } = require("../helper/portedTokens");
+
+const config = {
+  bsc: {
+    windmillContract: "0x0b374F3C618FF06583E7C4a1207bcaF22343737E",
+    tokenContract: "0x627E86E9eC832b59018Bf91456599e752288Aa97",
+    liquidityContract: "0x1fC3152de89b0c6c36F0d330b7Be369d6dDB219F",
+    vaultContract: "0x973Abe726E3e37bbD8501B2D8909Fa59535Babdd",
+  }
+}
+function tvl(chainName) {
+  return async ( timestamp, block, chainBlocks ) => {
+    const balances = {};
+    let transform;
+    switch(chainName) {
+        case "avax":
+           transform = await transformAvaxAddress();
+           break;
+         case "polygon":
+           transform =  await transformPolygonAddress();
+           break;
+      case "cronos":
+        transform =  await transformCronosAddress();
+        break;
+      default:
+        transform = await transformBscAddress();
+    };
+    const chain = config[ chainName ];
+    const totalDeposited = (await sdk.api.abi.call({
```

Comment:
does this return the amount of GALE tokens that are burnt and never can be retrieved? if so, sorry but It cant be counted towards tvl, if the tokens can be retrieved by the user, it should still count towards 'staking' not tvl